packages/happn-cluster/README.md

```diff
@@ -6,123 +6,94 @@
-        // certPath: 'path/to/cert'
-      }
-    },
-
-    // orchestrator sub-config (defaults displayed)
-    orchestrator: {
-      config: {
-        minimumPeers: 1,
-        replicate: ['*'],
-        stableReportInterval: 5000,
-        stabiliseTimeout: 120 * 1000 // 0 disables
-      }
-    },
-
-    // membership sub-config (defaults displayed)
-    membership: {
-      config: {
-        clusterName: 'happn-cluster',
-        seed: false,
-        seedWait: 0,
-        randomWait: 0,
-        // host: undefined, // defaults to first public IPv4 address
-        port: 56000,
-        // hosts: [],
-        joinTimeout: 2000,
-        pingInterval: 1000,
-        pingTimeout: 200,
-        pingReqTimeout: 600,
-        pingReqGroupSize: 3,
-        udp: {
-          maxDgramSize: 512
+    // name: undefined,  // defaults from happn service host:port (10-0-0-1_55000)
+    // host: '0.0.0.0', // happn service ip
+    // port: 57000,    // happn service port
+    // secure: true,  // to enable security
+    // announceHost: 'externally-visible-ip-or-hostname', // eg. when docker
+    services: {
+
+      // // security sub-config (to enable security)
+      // security: {
+      //   config: {
+      //     adminUser: {
+      //       username: '_ADMIN', // <---- leave this as _ADMIN
+      //       password: 'happn'
+      //     }
+      //   }
+      // },
+
+      // shared data plugin sub-config (defaults displayed)
+      data: {
+        config: {
+          datastores: [
+            //mongo is defaulted:
+            // {
+            //     name: 'mongo',
+            //     provider: 'happn-db-provider-mongo',
+            //     settings: {
+            //         collection: process.env.MONGO_COLLECTION || 'happn-cluster',
+            //         database: process.env.MONGO_DATABASE || 'happn-cluster',,
+            //         url: process.env.MONGO_URL || 'mongodb://127.0.0.1:27017',
+            //     },
+            //     isDefault: true,
+            // },
+          ]
+        }
+      },
+
+      // proxy sub-config (defaults displayed)
+      proxy: {
+        config: {
+          port: 55000,
+          host: '0.0.0.0',
+          allowSelfSignedCerts: false,
+          // timeout: 20 * 60 * 1000, // request socket idle timeout
+          // defer: false,
+          // keyPath: 'path/to/key',
+          // certPath: 'path/to/cert'
+        }
+      },
+
+      // orchestrator sub-config (defaults displayed)
+      orchestrator: {
+        config: {
+          clusterName: 'happn-cluster',
+          serviceName: 'happn-cluster-node',
+          deployment: 'Test-Deploy',
+          replicate: ['*'],
+          timing: {
+            keepaliveThreshold: 6e3,
+            stabiliseTimeout: 15e3,
+            keepAlive: 5e3,
+            memberRefresh: 5e3,
+            healthReport: 10e3
+          },
+          cluster: {
+            'happn-cluster-node': 1
+          },
+          minimumePeers: 1 //Deprecated
```

Comment:
document differently - not deprecated, this is the less fine-grained approach to the cluster config